CODE_OF_CONDUCT.md

```diff
@@ -14,22 +14,22 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
```

Comment:
Don't include unrelated changes in the PR. Remove this changes in this PR. Better send another